jujugui/static/gui/src/app/components/budget-table/row/row.js

```diff
@@ -97,7 +140,7 @@ YUI.add('budget-table-row', function() {
      @returns {Object} The plan form.
     */
     _generateChangePlan: function() {
-      if (!this.props.plansEditable) {
+      if (!this.props.plansEditable || this.state.plans.length === 0) {
```

Comment:
Is it possible that 'this.state.plans' could be null? We've been bitten by this elsewhere, so a little type safety check might be good here.